src/nccl_ofi_rdma.c

```diff
@@ -702,31 +699,23 @@ static inline int set_eager_copy_completed(nccl_net_ofi_rdma_req_t *req)
  */
 static inline int set_send_ctrl_completed(nccl_net_ofi_rdma_req_t *req)
```

Comment:
Comment above this function is incorrect, but now the function itself is no longer setting the control to be completed and is just freeing the request, so process_completions() should just do that and remove this function.